src/engine/strat_engine/backstore/metadata/static_header.rs

```diff
@@ -66,6 +67,15 @@ impl fmt::Display for StratisIdentifiers {
     }
 }
 
+impl<'a> Into<Value> for &'a StratisIdentifiers {
+    fn into(self) -> Value {
+        json!({
+            "pool_UUID": Value::from(self.pool_uuid.to_simple_ref().to_string()),
+            "device_UUID": Value::from(self.device_uuid.to_simple_ref().to_string())
```

Comment:
These should be `pool_uuid` and `device_uuid` if we're following conventions for JSON keys.